src/main/scala/org/dia/algorithms/mcc/MCSUtils.scala

```diff
@@ -18,189 +18,169 @@
 package org.dia.algorithms.mcc
 
 import java.util
-import scala.collection.mutable.ListBuffer
-import ucar.ma2.{ArrayInt, ArrayDouble, DataType}
-import ucar.nc2.{NetcdfFileWriter, Dimension, Attribute, Variable }
-import scala.reflect.ClassTag
-import scala.collection.JavaConversions._
+
+import ucar.ma2.{ArrayDouble, ArrayInt, DataType}
+import ucar.nc2.{Attribute, Dimension, NetcdfFileWriter, Variable}
+
+import scala.collection.JavaConverters._
```

Comment:
Nice use of JavaConverters.

The ordering can be fixed by swapping the import.scala... statements and import ucar.ma2... statements.

The group ordering is as follows :

java

scala,

everything else that's not spark or scispark, java..., or scala...

org.apache.spark....

org.dia....